<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrawSettings</class>
 <widget class="QWidget" name="DrawSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBoxDrawSelection">
         <property name="title">
          <string>Draw</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="checkBoxDrawPointSymbols">
            <property name="text">
             <string>Point Symbols</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxDrawPointIDs">
            <property name="text">
             <string>Point IDs</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxDrawEllipses">
            <property name="text">
             <string>Ellipses</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxDrawObservations">
            <property name="text">
             <string>Observations</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxDrawAxes">
            <property name="text">
             <string>Axes</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxSizeSelection">
         <property name="title">
          <string>Size</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="labelFontSize">
              <property name="text">
               <string>Font size</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="labelStrokeWidth">
              <property name="text">
               <string>Stroke width</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelSymbolSize">
              <property name="text">
               <string>Symbol size</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBoxFontSize"/>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBoxSymbolSize">
              <property name="enabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBoxStrokeWidth"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="labelEllipsesScale">
              <property name="text">
               <string>Ellipses scale</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBoxEllipsesScale"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxPointSymbols">
       <property name="title">
        <string>Point Symbols</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="2">
           <widget class="QComboBox" name="comboBoxConstrainedFill"/>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="comboBoxConstrainedSymbol"/>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="labelSymbol">
            <property name="text">
             <string>Symbol</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelPoints">
            <property name="text">
             <string>Points</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelConstrainedPoints">
            <property name="text">
             <string>Constrained</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelFixedPoints">
            <property name="text">
             <string>Fixed</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="comboBoxFreeSymbol"/>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBoxFixedSymbol"/>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="labelFill">
            <property name="text">
             <string>Fill</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelPointsFree">
            <property name="text">
             <string>Free</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QComboBox" name="comboBoxFixedFill"/>
          </item>
          <item row="3" column="2">
           <widget class="QComboBox" name="comboBoxFreeFill"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::Reset|QDialogButtonBox::RestoreDefaults</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
